train_trax/server/objects/Article.js

```diff
@@ -7,6 +7,11 @@ export default class Article {
 		return res.length === 0 ? [] : res;
 	}
 
+	static async getOrderedArticlesFromTeam(team) {
```

Comment:
This method name suggests that the articles are ordered somehow, but they're ordered defaultly.

Reply:
Also suggests that its from a team, when the parameter is a list of users.